src/document/DocumentCommandHandlers.js

```diff
@@ -1596,6 +1597,12 @@ define(function (require, exports, module) {
         ProjectManager.showInTree(MainViewManager.getCurrentlyViewedFile(MainViewManager.ACTIVE_PANE));
     }
 
+    /** Download selected file or folder structure **/
+    function handleFileDownload() {
+        var entry = ProjectManager.getSelectedItem();
+        ArchiveUtils.archiveWithPath(entry._path,function(){});
```

Comment:
Thanks for the feedback.
As I'm applying these changes, I noticed that `saveAs()` doesn't take a blobURL as a parameter.

![screen shot 2017-04-06 at 10 52 32 am](https://cloud.githubusercontent.com/assets/1103528/24760629/3fd1caf0-1ab7-11e7-9e0c-7a236cac388f.png)

![screen shot 2017-04-06 at 10 52 53 am](https://cloud.githubusercontent.com/assets/1103528/24760639/44c360a0-1ab7-11e7-84c5-8816f386713d.png)

Is there a step that I'm missing?